src/renderer/common/components/dialog/publicationInfos/publicationInfoContent.tsx

```diff
@@ -107,225 +108,225 @@ const Progression = (props: {
         // (Audiobooks, PDF, Divina, EPUB FXL and reflow ... page number vs. string types)
         try {
 
-        const isAudio = locatorExt.audioPlaybackInfo
-            // total duration can be undefined with badly-constructed publications,
-            // for example we found some LibriVox W3C LPF audiobooks missing duration property on reading order resources
-            && locatorExt.audioPlaybackInfo.globalDuration
-            && typeof locatorExt.locator.locations.position === "number"; // .progression is local to audio item in reading order playlist
-
-        const isDivina = r2Publication && isDivinaFn(r2Publication);
-        const isPdf = r2Publication && isPdfFn(r2Publication);
-
-        // locatorExt.docInfo.isFixedLayout
-        const isFixedLayout = r2Publication && // && !r2Publication.PageList
-            r2Publication.Metadata?.Rendition?.Layout === "fixed";
-
-        let txtProgression: string | undefined;
-        let txtPagination: string | undefined;
-        let txtHeadings: JSX.Element | undefined;
-
-        if (isAudio) {
-            const percent = Math.round(locatorExt.locator.locations.position * 100);
-            // const p = Math.round(100 * (locatorExt.audioPlaybackInfo.globalTime / locatorExt.audioPlaybackInfo.globalDuration));
-            txtProgression = `${percent}% [${formatTime(Math.round(locatorExt.audioPlaybackInfo.globalTime))} / ${formatTime(Math.round(locatorExt.audioPlaybackInfo.globalDuration))}]`;
-        } else if (isDivina) {
-            // console.log("----- ".repeat(100), divinaNumberOfPages, r2Publication?.Spine?.length);
-            let totalPages = (divinaNumberOfPages && !divinaContinousEqualTrue) ? divinaNumberOfPages : (r2Publication?.Spine?.length ? r2Publication.Spine.length : undefined);
-            if (typeof totalPages === "string") {
-                try {
-                    totalPages = parseInt(totalPages, 10);
-                } catch (_e) {
-                    totalPages = 0;
+            const isAudio = locatorExt.audioPlaybackInfo
```

Comment:
Why change a lot of code by adding a tabulation if the linting already pass and you do not modify these lines ? 

The diff become too heavy, difficult to check change

Reply:
IDE automatic linter is to blame on that one I think